src/data/forms.ts

```diff
@@ -69,6 +69,7 @@ export interface Form {
     beginText: ButtonText;
     changeText: ButtonText;
     confirmText: ButtonText;
+    backToTopText: ButtonText;
```

Comment:
we didn't intend to make this configurable in the backend, but rather pass it in the template down to the SDK constructor, see: 

https://github.com/open-formulieren/open-forms/blob/49fdc74dbcf14601f6f2158cef67d79e2d18acc6/src/openforms/forms/templates/forms/sdk_snippet.html#L4

and https://github.com/open-formulieren/open-forms/blob/49fdc74dbcf14601f6f2158cef67d79e2d18acc6/src/openforms/static/sdk-wrapper.mjs#L25

Reply:
I've change it to a data set parameter in the template